### PR TITLE
Create Project README

### PR DESCRIPTION
**Pull request by sarahkoop**
### Summary of changes

- Create project README based on RFC

### Checklist

- ~[ ] Added a changelog entry~

- @scannillo 
- @sarahkoop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,82 @@
...
+
+TBD until development progresses. We will use Rake, Fastlane, or some other tool for easy command-line build tasks.
+
+## Release Process
+
+This SDK follows [Semantic Versioning](https://semver.org/). The release process will be automated via GitHub Actions.
+
+## Analytics
+
+Client analytics will be collected via Lighthouse/FPTI.
```
Comment (jonathajones) [inline README.md]: Is this important to highlight? If the target audience for this readme is end merchants, do they need to know about it?

Comment (sarahkoop) [reply README.md]: This document is currently more of a mix of internal agreed upon tools, process, etc and the start of external facing documentation. I agree this info isn't needed for merchants, so if we don't think it's helpful internally either we can remove it.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,82 @@
...
+// STEP 4: Call checkout method
+cardClient?.checkoutWithCard(orderID: ORDER_ID, card: card) { (result, error) in
+    if (error != nil) {
+        // handle checkout error
+        return
+    }
+    
+    guard let orderID = result?.orderID else { return }
+    // Send orderID to your server to process the payment
+}
```
Comment (jonathajones) [inline README.md]: This snippet could be simplified if we use the `Result` type (even though this is just an example, i think this is clearer): 

```suggestion
cardClient?.checkoutWithCard(orderID: ORDER_ID, card: card) { result in
    switch result {
    case .success(let orderId):
      // Send orderID to your server to process the payment
    case .error(let error):
      // handle checkout error
    }
}
```

File: README.md (hunk 1 above)
Comment (jonathajones) [reply README.md]: That works! I'm fine leaving it as is if this is for our team as well

File: README.md (hunk 2 above)
Comment (scannillo) [reply README.md]: Good call! Let's definitely use the Swift `Result` type where we can.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,81 @@
...
+
+### Package Managers
+This SDK supports:
+
+* CocoaPods
+* Swift Package Manager
+
+### Languages
+
+This SDK supports Swift 5.1+. This SDK is written in Swift.
```
Comment (jonathajones) [inline README.md:17]: Should we also call out our minimum OS target here?

**Review by jonathajones — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sarahkoop) [reply README.md:17]: That seems redundant to me since it's stated a few lines above - but can add it again if we think it's helpful.

Comment (jonathajones) [reply README.md:17]: Whoops, missed it above. No need!

File: README.md [hunk 4]
```diff
@@ -1,5 +1,81 @@
 # PayPal iOS SDK
 
-![GitHub Actions Tests](https://github.com/paypal/iOS-SDK/workflows/Tests/badge.svg)
+Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payments in your iOS app.
```
Comment (jaxdesmarais) [inline README.md]: nit: can we update "alternative payments" => "alternative payment methods"? That appears to be the language we currently use in PayPal and Braintree docs for this.

**Review by jaxdesmarais — APPROVED** [review]
Just a small comment - looks good to me otherwise. Thanks for making these updates!

**Pull request merged**